src/components/Textfield/textfield.scss

```diff
@@ -19,6 +19,7 @@
     }
 
     &-text-area-show-count {
+      box-shadow: 0px 0px 8px rgba(106, 110, 128, 0.15);
```

Comment:
Can you move the rgba color to a variable?
Refer - https://stackoverflow.com/a/41265350/12402368

Because we use variables everywhere and its helpful to be used in other places.

Reply:
copied it from the previous code of show shadow 

 .contacto-input {
    border-radius: 0;
    box-shadow: 0px 0px 8px rgba(106, 110, 128, 0.15);
    border: 1px solid var(--border-color-1);
    height: 36px;

Reply:
We will need to update this in other places as well.. We'll do this later.